forms/jqm/elements/choiceexpanded.js

```diff
@@ -21,24 +21,35 @@ define(function (require) {
       return ChoiceElementView.prototype.remove.call(this);
     },
 
+    renderLabel: function () {
+      if (!this.$label) {
+        this.$label = $('<legend></legend>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
What if you just used `jQuery.contains()` directly? http://api.jquery.com/jQuery.contains/